pkgs/test_api/lib/src/frontend/spawn_hybrid.dart

```diff
@@ -89,7 +89,8 @@ final _transformer = StreamChannelTransformer<dynamic, dynamic>(
 ///
 /// **Note**: If you use this API, be sure to add a dependency on the
 /// **`stream_channel` package, since you're using its API as well!
-StreamChannel spawnHybridUri(uri, {Object? message, bool stayAlive = false}) {
+StreamChannel<dynamic> spawnHybridUri(uri,
```

Comment:
This change would be technically statically breaking... if we have other breakages anyways though.... 

```suggestion
StreamChannel<dynamic> spawnHybridUri(Object uri,
```

We could also do `Object?` instead but `null` isn't actually allowed and will fail at runtime...